src/commands/basic/ranking.js

```diff
@@ -64,7 +64,7 @@ class Ranking extends Command {
 
         if (topJokes) {
           for (const [i, jokes] of topJokes.entries()) {
-            const joke = config.messages.commands.joke.jokes[joke.id];
+            const joke = config.messages.commands.joke.jokes[jokes.id];
```

Comment:
En général quand on a une variable contenant un array, on lui mets un "s" à la fin. Donc là c'est ce qu'on a fait avec "topJokes". Mais ensuite quand on itère dans cet array, la variable courante on l'appelle comme l'array mais sans le "s".
Le truc c'est que la la variable n'a pas le même nom que l'array mais surtout elle a un "s" à la fin ^^ ce serai cool que tu la renommes.